rnd/autogpt_server/autogpt_server/util/process.py

```diff
@@ -13,6 +16,10 @@ class AppProcess(ABC):
     process: Optional[Process] = None
     set_start_method("spawn", force=True)
 
+    set_start_method("spawn", force=True)
+    configure_logging()
```

Comment:
Why here instead of in `run()`? When is this statement executed?

Reply:
this is the code path for running a process, so any code path should pass this, even tests

Reply:
Are you aware that that may actually break log capture in tests? Pytest installs its own log handlers and the config of those handlers can be changed using e.g. `-o log_level=DEBUG`. This only works for the first level of processes though, not for child processes.

Reply:
It seems to work for regular use, but I still think setup code like `configure_logging()` or `set_start_method(..)` shouldn't be run in a class definition like this. It makes the code non-reusable and harder to understand.